docs/wallet/index.md

```diff
@@ -302,3 +302,14 @@ With Zerion’s infrastructure you can build feature-rich real-time web3 apps, w
 - Homepage: [zerion.io](https://zerion.io/)
 - Platforms: Browser, iOS, Android
 - [Source Code](https://developers.zerion.io/reference/getting-started)
+
+---
+
+### [OKX Web3 Wallet](https://web3.okx.com/)
+
+Suppots more than 130 blockchains including CELO L2, allows to store, trade, analyze, earn and connect to web3 sites. The mobile version has its own browser to connect easier to web3 sites and has a tab to switch to the OKX exchange (although it is not mandatory to have an account in the exchange to use the wallet). It is well supported by WalletConnect and many web3 sites. The browser extension makes easier the development of web3 applications.
+
```

Comment:
Update to:

Supports over 130 blockchains, including CELO. Enables users to store, trade, analyze, earn, and connect to Web3 sites. The mobile app features a built-in browser for seamless Web3 access and includes a tab to switch to the OKX exchange (though creating an exchange account is not required to use the wallet). It’s fully compatible with WalletConnect and widely supported across Web3 platforms.